contracts/swaps/connectors/testnet/SwapsImplLocal.sol

```diff
@@ -25,11 +25,9 @@ contract SwapsImplLocal is State, ISwapsImpl {
         uint256 requiredDestTokenAmount)
         public
         returns (uint256 destTokenAmountReceived, uint256 sourceTokenAmountUsed)
-    {
+    {   
         require(sourceTokenAddress != destTokenAddress, "source == dest");
-        require(sourceTokenAddress != address(wethToken) && destTokenAddress != address(wethToken),
-            "WETH swaps not supported on testnet"
```

Comment:
is this ok?

Reply:
yes, because i adapted the TestWBTC (formerly TestWETH) contract to also allow arbitrary minting and burning.

Reply:
needed it for testing